src/swagger.json

```diff
@@ -0,0 +1,1142 @@
+                  "type": "object",
+                  "properties": {
+                    "message": {
+                      "type": "string",
+                      "example": "The item with the specified ID was not found."
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      "put": {
+        "summary": "Updates the item data with the specified ID",
+        "tags": ["Items"],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "description": "Item ID",
+            "schema": {
+              "type": "string",
+              "example": "6213921454b30d140000b395"
+            }
+          }
+        ],
+        "requestBody": {
+          "required": true,
+          "description": "One or more fields that needs to be updated",
+          "content": {
+            "application/json": {
+              "schema": {
+                "type": "object",
+                "properties": {
+                  "name": {
+                    "type": "string",
+                    "example": "Sofa"
+                  },
+                  "type": {
+                    "type": "string",
+                    "example": "Stationery"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "security": [
+          {
+            "authCookie": []
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful Operation",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "$ref": "#/components/schemas/Item"
+                }
+              }
+            }
+          },
+          "422": {
+            "description": "Invalid ID or unavailable item ID",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "message": {
+                      "type": "string",
+                      "example": "The item with the specified ID was not found."
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Unauthenticated or Unauthorized Request",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "message": {
+                      "type": "string",
+                      "example": "unauthorized to modify requested item: only item owner can modify"
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      },
+      "delete": {
+        "summary": "Updates the item data with the specified ID",
```

Comment:
this should be "Deletes the item data with the specified ID"

Reply:
Thank you for pointing this out 😁 
It happens a lot when copying and pasting from the other endpoints 😅

Reply:
By the way, instead of adding a comment for each change, you can press the `start a review` button and once you finish the review, go back to the top and press on finish review, add a general comment, and choose if you are requesting changes or approving and then submit them as one review

Reply:
yeah, I normally do that but for changes like grammar mistakes or wrong messages, I just leave a comment immediately :D